llvm/test/CodeGen/AMDGPU/select-flags-to-fmin-fmax.ll

```diff
@@ -1684,8 +1684,12 @@ define <4 x half> @v_test_fmin_legacy_ule_v4f16_nnan_nsz_flag(<4 x half> %a, <4
 ; GFX9-LABEL: v_test_fmin_legacy_ule_v4f16_nnan_nsz_flag:
 ; GFX9:       ; %bb.0:
 ; GFX9-NEXT:    s_waitcnt vmcnt(0) expcnt(0) lgkmcnt(0)
+; GFX9-NEXT:    v_pk_max_f16 v2, v2, v2
```

Comment:
This is all regressing, because the vector cases are custom. Essentially anywhere using isOperationLegal as an optimization heuristic should be using isOperationLegalOrCustom